tcl/file.tcl

```diff
@@ -175,6 +175,8 @@ proc ::file::Open_ {{fName ""} } {
   } else {
     if {$ext == ".si5" || $ext eq ""} {
       set dbType "SCID5"
+      # do not set a filename without extension in the recent file list
```

Comment:
This feels wrong.
What's the code that invoke ::file::Open with a wrong filename?